src/v8/v8.cc

```diff
@@ -279,10 +263,17 @@ bool V8::load(const std::string &code, bool allow_precompiled) {
   source_ = wasm::vec<byte_t>::make_uninitialized(code.size());
   ::memcpy(source_.get(), code.data(), code.size());
```

Comment:
Since parsing has been extracted to utils, we don't need to copy `source_` anymore before parsing... or maybe even at all.

Reply:
done.